pyvista/utilities/common.py

```diff
@@ -8,7 +8,7 @@
 
 
 def _coerce_pointslike_arg(
-    points: Union[NumericArray, VectorArray], copy: bool = True
+    points: Union[NumericArray, VectorArray], copy: bool = False
```

Comment:
I changed the default to False as IMO, copying is one of those things that should always default to false. @MatthewFlamm, I have no idea what consequences this might have

Reply:
I'm fine with this.  It is a private helper, just as long as we change all the usages that need `copy=True` here.

Some filters might take a points like argument and then modify it.  In this case, we always want to copy.  This is the motivation for having the option.  I'm neutral on what the best default is.

Reply:
Although the filters that it was implemented for specifically used `copy=False` which would further suggest this as the better default.

Reply:
Although the filters that it was implemented for specifically used `copy=False` which would further suggest this as the better default.